no-card.js

```diff
@@ -0,0 +1,24 @@
+import {
```

Comment:
That's an interesting approach. The name threw me off a little. 
I think EmptyCard might be more accurate, what do you think?

Reply:
Yes, I just used the first name crossed my mind :D